src/classes/UTIL_AbstractCommon_BATCH_TEST.cls

```diff
@@ -135,22 +135,18 @@ public class UTIL_AbstractCommon_BATCH_TEST {
     /*******************************************************************************************************************
      * @description Test Batch Execution when additional where clause is present (non LDV)
      */
-    // @isTest
-    // TODO - Fix this error: 
-    // System.UnexpectedException: No more than one executeBatch can be called from within a test method. 
-    // Please make sure the iterable returned from your start method matches the batch size, resulting in 
-    // one executeBatch invocation.
+    @isTest
     public static void executeAddtlWhereClause() {
         List<Account> accs = UTIL_UnitTestData_TEST.createMultipleTestAccounts(TEST_ACCOUNT_CNT, null);
         insert accs;
 
         Test.startTest();
         UTIL_AbstractCommon_MOCK tab = new UTIL_AbstractCommon_MOCK(10, 'Id = \'' + accs[0].Id + '\'');
-        tab.setTestChunkSize(10);
-        tab.setTestMaxNonLDVSize(10);
+        tab.setTestChunkSize(UTIL_AbstractCommon_BATCH.DEFAULT_LDV_CHUNK_SIZE);
+        tab.setTestMaxNonLDVSize(UTIL_AbstractCommon_BATCH.DEFAULT_MAX_NON_LDV_SIZE);
         tab.setExpectedResult(1);
 
-        Database.executeBatch(tab, 1);
+        Database.executeBatch(tab, 10);
```

Comment:
What does the change from 1 to 10 for the batch size do here if the condition is supposed to limit this to one record queried anyway. If the querying is working correctly, then a batch size of 1 should be fine because there can never be more than one record.

Reply:
Mostly I did it to make sure the batch would run blow up in a weird way if the batch returned more than one record.  It would still error when checking the expected result.  I can put it back though-- it is a good callout.  I am also going to set the testChunkSize to 1, and see about adding an assertion to make sure it is not running in ldv mode unexpectedly in the ones it should not be.